test/distributed/test_functional_api.py

```diff
@@ -384,23 +401,21 @@ def test_all_reduce(self):
         self.assertIsNone(x.grad)
 
 
-class TestMakeFx(TestCase):
+@skipIfHpu
+class TestMakeFx(MultiThreadedTestCase):
```

Comment:
Mind sharing the reason for changing the base test class?

Reply:
This was not required. 

I accounted for this change along with changes mentioned by @yifuwang in https://github.com/pytorch/pytorch/pull/138216#discussion_r1817188099, in the following commit: 7afa48037bfcdc0f2f8837cebcaefcfbf633de89

Thanks